internal/datastore/mysql/migrations/zz_migration.0002_namespace_tables.go

```diff
@@ -0,0 +1,50 @@
+package migrations
+
+// namespace max size: https://buf.build/authzed/api/file/main/authzed/api/v0/core.proto#L29
```

Comment:
it looks like the initial schema is missing some indexes that the pg driver has:

https://github.com/authzed/spicedb/blob/7c81b9c100aaddfe14781b125a627f4f745e1d1a/internal/datastore/postgres/migrations/add_reverse_index.go#L4-L5
https://github.com/authzed/spicedb/blob/7c81b9c100aaddfe14781b125a627f4f745e1d1a/internal/datastore/postgres/migrations/add_transaction_timestamp_index.go#L4

is there a reason to leave those out at this stage?